src/components/EmailListForm/EmailListForm.module.css

```diff
@@ -0,0 +1,29 @@
+input {
+  color: #2a2a2a;
+  border: none;
+}
+
+button,
+input {
+  padding: 1rem 2rem;
+  margin-top: .5rem;
+}
+
+button {
+  display: inline-block;
+  padding: 1rem;
+  margin-left: .5rem;
+  
+  border: none;
+  background-image: none;
+  background-color: #d43A69;
```

Comment:
this is likewhoa bright, let's make it that EDGI green color and save our main call-to-action pink for the mailing list signup